lib/smart_answer/calculators/state_pension_topup_calculator.rb

```diff
@@ -1,16 +1,35 @@
 module SmartAnswer::Calculators
   class StatePensionTopupCalculator
+    include ActiveModel::Model
+
     FEMALE_YOUNGEST_DOB = Date.parse('1953-04-05')
     MALE_YOUNGEST_DOB = Date.parse('1951-04-05')
     TOPUP_START_DATE = Date.parse('2015-10-12')
     TOPUP_END_DATE = Date.parse('2017-04-05')
     FEMALE_RETIREMENT_AGE = 62
     MALE_RETIREMENT_AGE = 65
 
-    def lump_sum_and_age(dob, weekly_amount, gender)
-      return [] if too_young?(dob, gender)
+    attr_accessor :date_of_birth
+    attr_accessor :gender
+    attr_accessor :weekly_amount
+
+    def initialize(attributes = {})
+      super
+      @gender ||= 'female'
```

Comment:
In the new version of this PR, I've moved the constructor definition (including the default value for `@gender`) from the commit titled "Use attributes vs arguments in lump_sum_and_age calculator method" (a8c67aadedb036b60ab68dfc735209886698c510) into the later commit titled "Use attributes vs arguments in too_young? calculator method" (c37cbc8cc7afd057458533df5463947fbf2dec6c) and added an explanatory paragraph to the latter's commit message.